geometry_msgs/Pose cam_pose
std_msgs/Float32MultiArray cam_intrinsics
sensor_msgs/Image depth
sensor_msgs/Image rgb
---
m2t2_ros/GraspCandidates candidates
